src/components/modals/AddressModal.js

```diff
@@ -89,11 +148,11 @@ export class AddressModal extends Component<AddressModalProps, AddressModalState
               <FormField
                 style={MaterialInputStyle}
                 value={uri}
-                onChangeText={this.updateUri}
+                onChangeText={this.onChangeTextDelayed}
                 error={''}
                 placeholder={s.strings.fragment_send_send_to_hint}
-                label={s.strings.fragment_send_send_to_hint}
-                onSubmit={() => this.props.onDone(uri)}
+                label={statusLabel}
+                onSubmit={() => this.props.onDone(cryptoAddress)}
```

Comment:
It looks like we will never send the URI to the onDone() Call back which is necessary if domain resolution does not succeed